UPGRADE.md

```diff
@@ -100,3 +100,82 @@
    +                       class: "%sylius.model.channel.class%"
    #           ...
    ```
+   
+1. Service and Alias Refactoring for [New Naming Convention](https://github.com/Sylius/Sylius/blob/2.0/adr/2024_10_03_services_naming_convention.md)
+   Including the following changes:
+
+#### Root configuration key
+
+   ```diff
+      - sylius_invoicing_plugin:
+      + sylius_invoicing:
+   ```
```

Comment:
Add also the note that this change makes changes to the resource's generated services and parameters

Reply:
Resource services should just go to the `Changed services` section so we don't overcomplicate things; the note about parameters is just not true, or at least does not relate to the root name. If any parameters have been changed, just make a new section and list them.